docs/nonlocal-networks.rst

```diff
@@ -0,0 +1,138 @@
+
+* When you are interacting with mainnet, make sure you verify all of the details of any transactions you sign/send before you send them. Brownie can't protect you from sending ETH to the wrong address, sending too much, etc. 
+* Always protect your private keys.  Don't leave them lying around unencrypted!
+
+Register with Infura
+========================
+Before you can connect to a non-local network, you need access to a remote Ethereum node that supports JSON RPC over HTTP.  `Infura <https://infura.io>`__ is one such option.  Once you register and create a project, Infura will provide you URLs with the API key that can be leveraged to access the given network.
+
+Non-local Network Configuration
+================================
+
+The connection settings for non-local networks must be defined in ``brownie-config.json``.
+
+First, for each network you want to configure, create a new section in the network.networks section as below:
+
+.. code-block:: javascript
+
+    "networks": {
+        .
+        .
+        "ropsten": {
+            "host": "http://ropsten.infura.io/v3/[yourInfuraApiKey]"
+        }
+        "rinkeby":...
+        .
+        .
+    }
+
+If you want to change the default network that brownie connects to, you need to update the network.default field as below:
+
+.. code-block:: javascript
+
+    "network": {
+        "default": "ropsten",
+        .
+        .
+    }
+
+Launching and Connecting to Networks
+====================================
+
+Using the CLI
+-------------
+
+Brownie will connect to whichever network is set as "default" in ``brownie-config.json``.  
+
+To connect to any other network that has been defined in ``brownie-config.json``, you must specify the `--network` flag when launching Brownie as below:
+::
+    $ brownie --network ropsten
+
+
+Each time Brownie is loaded, it will first attempt to connect to the ``host`` address to determine if the RPC client is already active.
+
+Using the ``brownie.netowrk`` methods
+-------------------------------------
+
+You can interact with any network defined in ``brownie-config.json`` programatically using the ``brownie.network.main`` methods.
+
+Connecting to a network:
+
+.. code-block:: python
+
+    >>> network.connect('ropsten')    
+    >>> network.is_connected()
+    True
+    >>> network.show_active()
+    'ropsten'
+
+Disconnecting from a network:
+
+.. code-block:: python
+
+    >>> network.disconnect()
+    >>> network.is_connected()
+    False
+
+Interacting with non-local networks
+==============================
+
+``brownie.module.rpc``
+--------
+
+The :ref:`rpc` module is unavailable when working with non-local networks.
+
+Accounts
+--------
+
+When loading an account for interacting with a non-local network, you must provide the private key when loading the account in order to be able to sign transactions or deploy contracts
```

Comment:
Would be good to clarify here that ``Accounts`` already includes all unlocked addresses as returned by `eth_accounts`. If the user is connected to their own private node there is no need to use `add`, `save`, or `load` - these methods are mostly useful when working on a hosted node such as Infura.

Reply:
Clarification added.